pkg/channel/testing/grpc_channel_server_mock.go

```diff
@@ -41,6 +43,7 @@ type MockServer struct {
 	BulkResponsePerPath            map[string]*runtimev1pb.TopicEventBulkResponse
 	initialized                    bool
 	mutex                          sync.Mutex
+	ValidateCloudEventExtension    *map[string]interface{}
```

Comment:
I don't why to use a pointer of map here.

Please help me understand the benefits of this usage